README.md

```diff
@@ -29,7 +29,7 @@ Install with pip:
 
     pip install mbtest
 
-(As usual, use of a [venv](https://docs.python.org/3/library/venv.html) or [virtualenv](https://virtualenv.pypa.io) is recommended.) Also requires [Mountebank](http://www.mbtest.org/) to have been installed:
+(As usual, use of a [venv](https://docs.python.org/3/library/venv.html) or [virtualenv](https://virtualenv.pypa.io) is recommended.) Also requires(optionally) [Mountebank](http://www.mbtest.org/) to have been installed:
```

Comment:
There does need to be an MB server _somewhere_, so let's not call it optional.

Reply:
fixed